buildlib/azure-pipelines-release.yml

```diff
@@ -5,52 +5,27 @@ trigger:
   tags:
     include:
       - v*
-pr:
-  - master
-  - v*.*.x
 
 resources:
   containers:
-    - container: centos7_cuda11
-      image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7-mofed5-cuda11:2
-    - container: centos8_cuda11
-      image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos8-mofed5-cuda11:2
-    - container: ubuntu16_cuda11
-      image: rdmz-harbor.rdmz.labs.mlnx/ucx/ubuntu16.04-mofed5-cuda11:3
-    - container: ubuntu18_cuda11
-      image: rdmz-harbor.rdmz.labs.mlnx/ucx/ubuntu18.04-mofed5-cuda11:3
-    - container: ubuntu20_cuda11
-      image: rdmz-harbor.rdmz.labs.mlnx/ucx/ubuntu20.04-mofed5-cuda11:3
+    - container: centos7
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7:4
 
-stages:
-  - stage: Prepare
-    jobs:
-      - job: CheckRelease
-        pool:
-          name: MLNX
-          demands:
-          - ucx_docker -equals yes
-        steps:
-          - checkout: self
-            fetchDepth: 100
-            clean: true
-
-          - bash: |
-              set -eE
-              source ./buildlib/az-helpers.sh
-              set -x
-              check_release_build $(Build.Reason) $(Build.SourceVersion) "AZP/RELEASE: "
-            name: Result
-            displayName: Check build condition
 
+stages:
   # Create an empty draft to avoid race condition in distro releases
-  - stage: GitHubDraft
-    condition: eq(dependencies.Prepare.outputs['CheckRelease.Result.Launch'], 'True')
-    dependsOn: Prepare
+  - stage: GitHubRelease
+    variables:
+      ${{ if eq(variables['Build.Reason'], 'ResourceTrigger') }}:
+        artifact_name: ucx-${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}-centos7-mofed5.tar.bz2
+      ${{ if eq(variables['Build.Reason'], 'IndividualCI') }}:
+        artifact_name: ucx-${{ replace(variables['Build.SourceBranch'], 'refs/tags/', '') }}-centos7-mofed5.tar.bz2
+      ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
```

Comment:
no need to run release pipeline in case of PR

Reply:
it will not run, it just copy paste from release job

Reply:
I'd remove it then